athletics.lic

```diff
@@ -66,6 +66,15 @@ class Athletics
     stow_athletics_items
   end
 
+  def outdoorsmanship_waiting(n)
+    start = Time.now
+    walk_to(@outdoorsmanship_rooms.sample) unless @outdoorsmanship_rooms.empty?
+    stow_athletics_items
+    wait_for_script_to_complete('outdoorsmanship', [n])
+    pause (n*15) - (Time.now - start)
```

Comment:
Since this script is about athletics training, some folks (👋) might not realize there's potentially a pause after the script runs when nothing is hapenning. Suggesting we put an echo to clarify.
```suggestion
    time_to_wait = (n*15) - (Time.now - start)
    if time_to_wait > 0
      echo "Almost ready, need to wait #{time_to_wait.to_i} seconds before resume athletics training"
      pause time_to_wait
    end
```